xmppserver/src/main/java/org/jivesoftware/openfire/muc/spi/MultiUserChatServiceImpl.java

```diff
@@ -551,6 +551,9 @@ else if (this.iqHandlers != null) {
      */
     private void sendErrorPacket( Packet packet, PacketError.Condition error, String message )
     {
+        if (packet.getError() != null) {
+            Log.debug("Avoid generating an error in response to a stanza that itself has an error (to avoid the chance of entering an endless back-and-forth of exchanging errors). Suppress sending an {} error (with message '{}') in response to: {}", error, message, packet);
+        }
```

Comment:
Does this need a return statement to avoid sending the packet?

Reply:
Yes, thank you.